uniflow/op/model/lm/model_server.py

```diff
@@ -10,6 +10,8 @@
 from functools import partial
 from typing import Any, Dict, List, Optional
 
+from transformers import AutoModelForCausalLM, AutoTokenizer, pipeline
```

Comment:
You should import in class level like before. Otherwise, this will break unittest.